build/release

```diff
@@ -6,8 +6,8 @@ if [ $# -ne 2 ]; then
 fi
 old_version=$1
 new_version=$2
-for file in configure.ac doc/api/common/version.md doc/index.md \
-            include/measurement_kit/common/version.h \
+for file in configure.ac doc/api/common/version.md doc/index.md                \
+            include/measurement_kit/common/version.h build/spec/mk             \
```

Comment:
Also update the version number in `build/spec/mk` (not used yet, but `#correctness`)